acceptance-tests/lib/TestState.ts

```diff
@@ -49,16 +49,26 @@ export class TestState {
         getInitPromptSequence(this.getPAK())
       );
 
+      await this.cli.executeWithTestConfig([
+        'config',
+        'set',
+        '--allowUsageTracking=false',
+      ]);
+
       this.parsedYaml = yaml.load(
         readFileSync(this.getTestConfigPath(), 'utf8')
       );
     } catch (e) {
       console.error(e);
-      // @ts-expect-error TypeScript thinks the cause doesn't exist
       throw new Error('Failed to initialize CLI config & authentication', {
         cause: e,
       });
     }
+
+    // @ts-expect-error Non-existent field according to TS
+    if (this.parsedYaml?.allowUsageTracking) {
+      throw new Error('Usage tracking should be disabled');
+    }
```

Comment:
I debated not adding this, but I think it might be worthwhile to fail the tests in the event usage tracking gets reenabled so we can quickly address it and prevent a sudden usage spike in the data.

Reply:
I think that's wise 👍